docs/en/latest/back-end-e2e.md

```diff
@@ -82,23 +82,25 @@ This document describes how to use E2E test locally.
 
 1. [install docker-compose](https://docs.docker.com/compose/install/)
 
-**NOTE:** In order to run docker compose locally, please change the values of `listen.host` and `etcd.endpoints` within `./api/conf/conf.yaml` as follows:
-
-```sh
-listen:
-   host: 0.0.0.0
-   port: 9000
-etcd:
-   endpoints:
-     - 172.16.238.10:2379
-     - 172.16.238.11:2379
-     - 172.16.238.12:2379
-```
+   **NOTE:** In order to run docker compose locally, please change the values of `listen.host` and `etcd.endpoints` within `./api/conf/conf.yaml` as follows:
+
+   ```sh
+   listen:
+      host: 0.0.0.0
+      port: 9000
+   etcd:
+      endpoints:
+        - 172.16.238.10:2379
+        - 172.16.238.11:2379
+        - 172.16.238.12:2379
+   ```
 
 2. Use `docker-compose` to run services such as `manager-api`, `apisix`, `etcd` and `upstream-node`, run the command.
 
    ```sh
    cd /(Your apisix-dashboard folder path)/api/test/docker
+   # Download the apisix dockerfile
+   curl -o Dockerfile-apisix https://raw.githubusercontent.com/apache/apisix-docker/master/alpine/Dockerfile
```

Comment:
Just want to know why we need to download the apisix dockerfile.

Reply:
The docker-compose file needs to build the apisix image from a dockerfile which is currently present at apisix-docker repository. For more details please go through issue #1581 . Thanks